Samples/kotlin/src/main/res/values/strings.xml

```diff
@@ -13,16 +13,16 @@
 
     <!-- Demos -->
     <string name="demo_load_building_title">Load Building</string>
-    <string name="demo_load_building_description">Loads a building on a Google Map</string>
+    <string name="demo_load_building_description">Load and display map</string>
 
     <string name="demo_load_building_no_map_title">Load Building without map</string>
     <string name="demo_load_building_no_map_description">Loads and lists building data without a map</string>
 
     <string name="demo_bluedot_title">Bluedot Location</string>
-    <string name="demo_bluedot_description">Show your location on the map using Phunware APIs</string>
+    <string name="demo_bluedot_description">Display current location on map</string>
 
     <string name="demo_location_modes_title">Location Modes</string>
-    <string name="demo_location_modes_description">Experience the different behavior of each location mode</string>
+    <string name="demo_location_modes_description">Different ways for the camera to follow the blue dot</string>
```

Comment:
Is this what iOS has in their description now for location modes?

Reply:
yes. kinda odd.
![image](https://user-images.githubusercontent.com/4603414/67526628-060d1e00-f66a-11e9-8901-3fc29a808e11.png)

Reply:
ok :)